debian/control

```diff
@@ -1,53 +1,19 @@
 Source: pop-plymouth-theme
 Section: x11
 Priority: optional
-Maintainer: Jeremy Soller <jeremy@system76.com>
-Uploaders: Ian Santopietro <ian@system76.com>
-Build-Depends: debhelper (>= 10), imagemagick
-Standards-Version: 3.9.8
-Homepage: https://github.com/system76/pop-plymouth-theme
+Maintainer: 13r0ck <bnr@tuta.io>
+Build-Depends: debhelper (>= 10.5.1)
+Standards-Version: 4.5.0
 
-Package: plymouth-theme-pop-basic
+Package: pop-plymouth-theme
 Architecture: all
-Depends: ${misc:Depends}, plymouth, plymouth-label, plymouth-themes, pop-fonts
+Depends: ${misc:Depends}, plymouth, plymouth-label
 Provides: plymouth-theme
-Breaks: pop-default-settings (<< 2)
-Replaces: pop-default-settings (<< 2)
-Description: graphical boot animation and logger - pop-basic theme
+Replaces: plymouth (<< 0.8.1-1~)
```

Comment:
This line seems a bit odd since there wasn't a `Replaces: plymouth` line in the previous version.

`Replaces:` is used when a package provides files that where previously provided by a different package, so the upgrade won't fail due to file conflicts. If that's the case here, I think `Breaks:` is also necessary. Or did you mean something else?

(`.deb` package relationships can get complicated.)

Reply:
That line was just pulled from elementry's packaging. So I honestly don't know :grimacing:

Reply:
Then they probably previously packaged their own theme as as part of the `plymouth` package, before splitting it off. If we haven't done that it wouldn't be necessary on Pop.